src/src/Static/WebStaticData.cpp

```diff
@@ -26,7 +26,6 @@ void serve_CSS() {
   #ifndef WEBSERVER_CSS
   addHtml(F("<link"));
   addHtmlAttribute(F("rel"), F("stylesheet"));
-  addHtmlAttribute(F("type"), F("text/css"));
```

Comment:
Not sure about removing this.
Is it that browsers will render it correct anyway, or is it according to the standards not having it?